### ISSUE TITLE
support for Angular 14 ?

### ISSUE THREAD

**Issue by ErendisNoob** [Enngage/ngx-captcha#132]
Hello,

ngx-captcha is not compatible with angular 14 yet ? 
i have conflict with npm when i try to install it
i have force it and it's work well in the code but i have fails with NPM every time i try to install something else because of the conflict.

**Comment by hanschrome** [comment]
Hello @ErendisNoob, I'm working on it. I got an error with "ng build" due to a wrong call, I added the merge request with the fix for that. Once I update the dependencies and I test it, I will add another pull request for Angular 14 support.

**Comment by assefvisic** [comment]
Hello :)

Any news on when the PR will be accepted to support angular 14 ?

Regards

**Comment by Enngage** [comment]
Hey, yeah, I'll try looking into it soon. Very busy at the moment :)

**Comment by ViktorHnativ** [comment]
Thanks we wait on it 👍

**Comment by ViktorHnativ** [comment]
Any news?

**Comment by Heatmanofurioso** [comment]
Hi, any news on this PR?
It's probably impacting a big set of the community using the dependency

**Comment by Rocket18** [comment]
Hi, any news?

**Comment by Heatmanofurioso** [comment]
I've made an updated PR for V14 support

**Comment by Enngage** [comment]
Sorry for the wait ;) I've just published an update.

**Comment by Snowy127** [comment]
Which version of ngx-captcha is compatible with Angular 14.0.0?
